sql-cloudant/src/test/scala/org/apache/bahir/cloudant/TestUtils.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.bahir.cloudant
+
+import java.io.File
+
+object TestUtils {
+  // List of test databases to create from JSON flat files
+  val testDatabasesList: List[String] = List(
+    "n_airportcodemapping",
+    "n_booking",
+    "n_customer",
+    "n_customersession",
+    "n_flight",
+    "n_flight2",
+    "n_flightsegment"
+  )
+
+  // Set CouchDB/Cloudant host, username and password for local testing
+  private val host = System.getenv("DB_HOST")
```

Comment:
@ckadner I can do that or I can do `CLOUDANT_HOST`.  Eventually I'd like to have `COUCHDB_HOST` as well.